pull/context.go

```diff
@@ -220,10 +220,23 @@ type Reviewer struct {
 }
 
 type Collaborator struct {
-	Name       string
+	Name        string
+	Permissions []CollaboratorPermission
+}
+
+func (c *Collaborator) HasExactPermission(p Permission, viaRepo bool) bool {
```

Comment:
Is this used anywhere?

Reply:
Good catch, I thought I was going to use this then ended up doing something else